standard/clause_specification_text.adoc

```diff
@@ -468,7 +468,7 @@ Its general structure is:
 //#### 6.1.1. Axis Objects
 ===== Axis Objects
 
-- An axis object MUST have either a `"values"` member or, as a compact notation for a regularly spaced numeric axis, all the members `"start"`, `"stop"`, and `"num"`.
+- An axis object MUST have either a `"values"` member or, as a compact notation for a regularly spaced numeric axis,  have the members `"start"`, `"stop"`, and `"num"`.
```

Comment:
I think the word "all" is important. How about "have all the members..."

Reply:
agreed